snippets/categories/ptn.mdx

```diff
@@ -0,0 +1,8 @@
+<details><summary>Piratethenet</summary>
+
+- 2160P
+- 1080P
+- 720P
+- WebRip
```

Comment:
Is there only WebRip and then no other category such as WEB WEBDL or disc categories?

Reply:
Yep. This is what I've gathered from the announces so far.